packages/e2e-tests/specs/editor/plugins/block-variations.test.js

```diff
@@ -1,191 +0,0 @@
-	searchForBlock,
-	pressKeyWithModifier,
-	openDocumentSettingsSidebar,
-	togglePreferencesOption,
-	toggleMoreMenu,
-	canvas,
-} from '@wordpress/e2e-test-utils';
-
-describe( 'Block variations', () => {
-	beforeAll( async () => {
-		await activatePlugin( 'gutenberg-test-block-variations' );
-	} );
-
-	beforeEach( async () => {
-		await createNewPost();
-	} );
-
-	afterAll( async () => {
-		await deactivatePlugin( 'gutenberg-test-block-variations' );
-	} );
-
-	const expectInserterItem = async ( blockTitle ) => {
-		const inserterItem = await page.$x(
-			`//button[contains(@class, 'block-editor-block-types-list__item')]//span[text()="${ blockTitle }"]`
-		);
-		expect( inserterItem ).toBeDefined();
-		expect( inserterItem ).toHaveLength( 1 );
-	};
-
-	test( 'Search for the overridden default Quote block', async () => {
-		await searchForBlock( 'Quote' );
-
-		expect( await page.$( '.editor-block-list-item-quote' ) ).toBeNull();
-		expectInserterItem( 'Large Quote' );
-	} );
-
-	test( 'Insert the overridden default Quote block variation', async () => {
-		await insertBlock( 'Large Quote' );
-
-		expect(
-			await canvas().$(
-				'.wp-block[data-type="core/quote"] blockquote.is-style-large'
-			)
-		).toBeDefined();
-	} );
-
-	test( 'Insert the Large Quote block variation with slash command', async () => {
-		await insertBlock( 'Paragraph' );
-
-		await page.keyboard.type( '/large' );
-		await page.keyboard.press( 'Enter' );
-
-		expect(
-			await canvas().$(
-				'.wp-block[data-type="core/quote"] blockquote.is-style-large'
-			)
-		).toBeDefined();
-	} );
-
-	test( 'Search for the Paragraph block with 2 additional variations', async () => {
-		await searchForBlock( 'Paragraph' );
-
-		expectInserterItem( 'Paragraph' );
-		expectInserterItem( 'Success Message' );
-		expectInserterItem( 'Warning Message' );
-	} );
-
-	test( 'Insert the Success Message block variation', async () => {
-		await insertBlock( 'Success Message' );
-
-		const successMessageBlock = await canvas().$(
-			'.wp-block[data-type="core/paragraph"]'
-		);
-		expect( successMessageBlock ).toBeDefined();
-		expect(
-			await successMessageBlock.evaluate( ( node ) => node.innerText )
-		).toBe( 'This is a success message!' );
-	} );
-	test( 'Pick the additional variation in the inserted Columns block', async () => {
-		await insertBlock( 'Columns' );
-
-		const fourColumnsVariation = await canvas().waitForSelector(
-			'.wp-block[data-type="core/columns"] .block-editor-block-variation-picker__variation[aria-label="Four columns"]'
-		);
-		await fourColumnsVariation.click();
-		expect(
-			await canvas().$$(
-				'.wp-block[data-type="core/columns"] .wp-block[data-type="core/column"]'
-			)
-		).toHaveLength( 4 );
-	} );
-	// @see @wordpres/block-editor/src/components/use-block-display-information (`useBlockDisplayInformation` hook).
-	describe( 'testing block display information with matching variations', () => {
-		beforeEach( async () => {
-			await togglePreferencesOption(
```

Comment:
Sorry if I missed it but I didn't see this in the migrated code. Do we not need this anymore? What was it for?

Reply:
The breadcrumbs are enabled by default. If any other test disables it, I think it should enable it back after the tests.

Reply:
Cool 👍 Thanks for the explanation!